### DISCUSSION TITLE
plugin-google-gtag  - including google-analytics.com

### DISCUSSION THREAD

**Discussion by markharrison** [facebook/docusaurus#6425] [Q&A] [answered]
I am using the  @docusaurus/plugin-google-gtag ^2.0.0-beta.14  plugin

But my deployed HTML contains ...

```
<link rel="preconnect" href="https://www.google-analytics.com">
<link rel="preconnect" href="https://www.googletagmanager.com">
```

My understanding of using gtag is I dont need the first link /script to www.google-analytics.com  ... just the second to www.googletagmanager.com.

I assume its the default Google Analytics plugin - can i disable it somehow ?

thx

**Comment by slorber** [comment, marked as answer]
As far s I understand it, you need both:

- we'll load this script: https://www.googletagmanager.com/gtag/js?id=UA-141789564-1
- and this script will load https://www.google-analytics.com/analytics.js

At least it's what I see on the Docusaurus website using GTAg

So pre-connecting to both seems useful
